vmm/src/vm.rs

```diff
@@ -1504,6 +1538,126 @@ impl Vm {
         }
     }
 
+    #[cfg(feature = "tdx")]
+    fn configure_tdx(&mut self) -> Result<()> {
```

Comment:
Nit: I would rename that one to `tdx_init`, in order to more clearly match the underlying ioctl.